### ISSUE TITLE
Remove option to set solr hostname, port and core TTW.

### ISSUE THREAD

**Issue by tisto** [collective/collective.solr#51]
Solr is aimed at larger deployments. Allowing to set server configuration TTW is superfluous. ZCML configuration should be the only way to configure solr.

If people care much about TTW configuration and are willing to work on that, an alternative would be to display the zcml settings on the control panel. Though, personally I don't think this is worth the effort.

**Comment by loechel** [comment]
@tisto you are right that Solr aims on larger deployments, but as for the very large deployments hosting several Plone instances within one zope cluster is not uncommon. If all instances needs to use the same Solr Server and Core that would be a restriction that I don't like. 

What I could agree on it to say, if a ZCML configuration exists, that should be shown in the control panel but there should be an option to overwrite this TTW. #12 sounds like that might be complicated as our solr implementation seems to share Connections Pooling in Zope.

What I would also like to see is the possibility to have a Solr-Master (write to) and  Solr-Slave / SolrCloud-Cluster with multiple Read-only solr-Instances usable and configurable. 

I would jump in to make both happen (new TTW-Style and multiple backends) if all of us agree on the way to do this. This would be an overlap with #50 that sould be solved in the same cleanup as a predecessor.

**Comment by tisto** [comment]
@loechel I always forget that people put multiple instances into one zope. :)

Ok, then let's move the control panel to plone.app.registry. We can then grey out the host/port/core if this has been set in zcml. This way we can fully support both use cases.

**Comment by tisto** [comment]
This should be implemented after #50 has been finished.

**Comment by loechel** [comment]
that seams like a good solution in the first place. let us do it like that.

**Comment by tomgross** [comment]
@tisto @loechel Anyone working on this? This is no blocker for a 6.0 release right?
